tox.ini

```diff
@@ -9,9 +9,9 @@ deps =
     pytest-cov
     flake8
 commands =
-    python setup.py check --strict
-    flake8 xsnippet_api/ tests/
-    py.test tests/ --cov --cov-append
```

Comment:
Hmm, don't you change the path here from `tests/` to `xsnippet_api/`? I thought, we had to path the base dir with tests.